webservice/src/main/java/com/hortonworks/registries/webservice/RegistryApplication.java

```diff
@@ -90,6 +87,33 @@ public void run(RegistryConfiguration registryConfiguration, Environment environ
 
     }
 
```

Comment:
Considering UGI is needed for only Authorization, can we move this code to RangerAuthorizer class instead? This should not be even called out when authorization is not enabled but security is on.
https://github.com/apache/ranger/blob/d6e0e9af2446d023e68df1e21221d8bf1190615d/plugin-kafka/src/main/java/org/apache/ranger/authorization/kafka/authorizer/RangerKafkaAuthorizer.java#L162

Reply:
The code cannot be moved to RangerAuthorizer. It is used not only for Kerberos authentication. UserGroupinformation is also used for user2groups mapping.